pkg/provision/vsphere.go

```diff
@@ -183,7 +183,7 @@ func createMaster(platform *platform.Platform) (types.Machine, error) {
 
 func createWorker(platform *platform.Platform, nodeGroup string) (types.Machine, error) {
 	if nodeGroup == "" {
-		for k, _ := range platform.Nodes {
+		for k := range platform.Nodes {
```

Comment:
linter prefers that when iterating over just the keys that the (optional) value be left out rather than _'ed